tests/InstrEngineTests/InstrEngineTests/NativeMethods.cs

```diff
@@ -11,6 +13,6 @@ internal class NativeMethods
         public static extern int GetComPlusPackageInstallStatus();
 
         [DllImport("kernel32.dll")]
-        public static extern int SetComPlusPackageInstallStatus(int flag);
+        public static extern bool SetComPlusPackageInstallStatus(int flag);
```

Comment:
>bool [](start = 29, length = 4)

?

Reply:
```SetComPlusPackageInstallStatus``` returns a BOOL so to not confuse the return type with HRESULT I updated the signature